packages/tasks/src/tasks/feature-extraction/inference.ts

```diff
@@ -0,0 +1,20 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+
+/**
+ * Inputs for Feature Extraction inference
+ */
+export interface FeatureExtractionInput {
```

Comment:
Btw not sure how to communicate about it, but we should consider at one point renaming to "Embedding" as that's the name used by the community nowadays. This endpoint retrieves sentence embeddings if it's a ST repo, or contextualized word embeddings if it's a TF repo

Reply:
How would we proceed to do this?